source/event_stream_rpc_server.c

```diff
@@ -361,36 +380,31 @@ static void s_on_accept_channel_shutdown(
 
 static void s_on_server_listener_destroy(struct aws_server_bootstrap *bootstrap, void *user_data) {
     (void)bootstrap;
-    struct aws_event_stream_rpc_server_listener *listener = user_data;
+    struct aws_event_stream_rpc_server_listener *server = user_data;
```

Comment:
Trivial: let's name this `server_listener` for clarity in its use across this file.